src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilterTest.java

```diff
@@ -420,10 +420,22 @@ public void testInvalidCheckFormat() throws Exception {
     @Test
     public void testAcceptNullLocalizedMessage() {
         final SuppressWithNearbyCommentFilter filter = new SuppressWithNearbyCommentFilter();
-        final TreeWalkerAuditEvent auditEvent = new TreeWalkerAuditEvent(null, null, null, null);
+        final FileContents contents =
+                new FileContents("filename", "//SUPPRESS CHECKSTYLE ignore");
+        contents.reportSingleLineComment(1, 0);
+        final TreeWalkerAuditEvent auditEvent =
+                new TreeWalkerAuditEvent(contents, null, null, null);
         assertTrue("Filter should accept null localized message", filter.accept(auditEvent));
     }
 
+    @Test
+    public void testAcceptNullFileContents() {
+        final SuppressWithNearbyCommentFilter filter = new SuppressWithNearbyCommentFilter();
+        final TreeWalkerAuditEvent auditEvent = new TreeWalkerAuditEvent(null, null,
```

Comment:
Too much nulls in arguments, please make variable `content = null` and use it in constructor.

Please apply the same for all other test method you changed